third_party/ruby/debug_counter.h.patch

```diff
@@ -0,0 +1,11 @@
+--- ruby-2.6.3/debug_counter.h	2018-12-13 17:10:15.000000000 -0800
++++ ruby-2.6.3-dirty/debug_counter.h	2019-07-29 23:15:26.000000000 -0700
+@@ -249,7 +249,7 @@
+ 
+ enum rb_debug_counter_type {
+ #define RB_DEBUG_COUNTER(name) RB_DEBUG_COUNTER_##name,
+-#include __FILE__
++#include "debug_counter.h"
+     RB_DEBUG_COUNTER_MAX
+ #undef RB_DEBUG_COUNTER
+ };
```

Comment:
What's the story with this patch?

Reply:
This is where they were doing file-level recursion through CPP. The problem was that it was resolving to a relative name in the sandbox, that would then fail to be included. Changing it to the actual name of the file resolved the issue.